APIWorking.js

```diff
@@ -1,65 +1,80 @@
-                  //Add search results to the text of the new div
-                  let textContent = document.createTextNode(`${resultsTitle} (${resultsType}, ${resultsYear})`);
-                  const newDiv = document.getElementById("main").appendChild(div);
-                  let poster = res.Search[i].Poster;
-                  let img = document.createElement("img");
-                  let linebr = document.createElement("br")
-                  img.src = poster;
-                  img.title = "movie poster (broken image link)";
-                  newDiv.appendChild(textContent)
-                  newDiv.appendChild(linebr)
-                  newDiv.appendChild(img)
-                  document.getElementById("main").appendChild(newDiv);
-                  newDiv.classList.add("results")
-                  
-                  // Console log test to make sure it's working:
-                  // console.log(`Title: ${resultsTitle} Type: ${resultsType} Year: ${resultsYear}`)
-                }
-              });
-            } else {
-                document.getElementById("main").innerText = "Please enter 3 or more characters!"
-            }};
-        }   
-          });
\ No newline at end of file
+  .getElementById("test")
+  .addEventListener("click", function fetchTest() {
+    //remove prior results if any
+    const node = document.getElementById("main");
+    node.querySelectorAll('*').forEach(n => n.remove());
+    //set up variables for search results, with url snippets
+    const title = "&s=" + document.getElementById("searchBox").value;
+    const radios = document.getElementsByName("type");
+    const year = `&y=${document.getElementById("yearDropdown").value}`
+    //make type equal to the checked radio button.
+    for (var radio of radios) {
+      if (!radio.checked) {
+        continue;
+      }
+
+      if (title.length < 5) {
+        document.getElementById("main").innerText = "Please enter 5 or more characters!"
+        return
+      }
+
+      const type = "&type=" + radio.value;
+      document.getElementById("main").innerText = ""
+      //run search with user inputs
+      fetch("http://www.omdbapi.com/?apikey=5d6248fb" + title + type + year)
+        //log results
+        .then(function (response) {
+          console.log(response);
+          //if response is NOT ok show response status
+          if (!response.ok) {
+            console.log(response.status);
+          }
+          return response.json();
+        })
+        .then(function (res) {
+          const resultLength = res.totalResults;
+          let resultsText = `Showing 10 of ${resultLength} results:` 
+
+          if (!resultLength) {
+            resultsText = 'No movies found'
+          }
+
+          document.getElementById("main").innerText = resultsText;
+
+          //TODO Could use a forEach instead of a for loop? Look into later
+          res.Search.forEach((movie) => {
+            const { Title, Type, Year, Poster } = movie
```

Comment:
The standard convention is that the only names that start with an uppercase letter should be classes, so it might be better to alias these guys to be all lower case like so:
```suggestion
            const { Title: title, Type: type, Year: year, Poster: poster } = movie
```